opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -2485,143 +2485,258 @@ bool CpGrid::adapt(const std::vector<std::array<int,3>>& cells_per_dim_vec,
     // - Define GlobalIdMapping (cellMapping, faceMapping, pointMapping required per level)
     // - Define ParallelIndex for overlap cells and their neighbors
     if(comm().size()>1) {
+        lgrCommunication(assignRefinedLevel,
+                         cells_per_dim_vec,
+                         lgr_with_at_least_one_active_cell);
+    }
+
+
+    // Print total amount of cells on the adapted grid
+    Opm::OpmLog::info(std::to_string(markedElem_count) + " elements have been marked (in " + std::to_string(comm().rank()) + " rank).\n");
+    Opm::OpmLog::info(std::to_string(levels)  + " (new) refined level grid(s) (in " + std::to_string(comm().rank()) + " rank).\n");
+    Opm::OpmLog::info(std::to_string(cell_count)  + " total cells on the leaf grid view (in " + std::to_string(comm().rank()) + " rank).\n");
+
+    return preAdapt();
```

Comment:
Are you sure this should be here?
This is the adapt method.

Reply:
Fixed now!